core/tests/puppeteer-acceptance-tests/utilities/user/super-admin.ts

```diff
@@ -285,11 +285,16 @@ export class SuperAdmin extends BaseUser {
    */
   async expectRoleToHaveAssignedUsers(users: string[]): Promise<void> {
```

Comment:
What if an error is encountered in this function? There's no throw new Error statement.

Reply:
Yes, fixing that too. Updated the description and the title accordingly. 
Also, waitForFunction() will throw a timeout error. However, as this might not clear enough, so, added a throw statement. 

Thanks.